### DISCUSSION TITLE
Hook Request (back office new order creation)

### DISCUSSION THREAD

**Discussion by musaffar-patel** [PrestaShop/PrestaShop#41070] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Describe the solution you'd like

Hi,

Would be very useful to have a hook which allows modules to display content in the new order creation form in the back office.

At the moment, I have to modify the file
adminps/themes/default/template/controllers/orders/form.tpl

and add the following code where I wish to display the module content:

		<div class="form-group">
			<div class="checkbox col-lg-9 col-offset-3">
				**{hook h='AdminOrder'}**
			</div>
		</div>
Or I have to ask the customer to make this modification and I desperately try to avoid making changes to core files.

Unless a hook is already available and I am missing it?

Kind Regards
Musaffar

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by Hlavtox** [comment]
@musaffar-patel No hooks there man, we need to add them. There are no hooks for carrier eiher - https://github.com/PrestaShop/PrestaShop/issues/25336.

**Comment by hibatallahAouadni** [comment]
Hello @musaffar-patel 

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. 

If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know! 

Thank you

**Comment by musaffar-patel** [comment]
ideally a simple hook in the file below would be very useful:
/src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/shipping.html.twig
